pom.xml

```diff
@@ -321,6 +340,16 @@ limitations under the License.
         <artifactId>lucene-spatial-extras</artifactId>
         <version>${version.lib.lucene}</version>
       </dependency>
+      <dependency>
```

Comment:
I don't think we need this dependency anymore because we use the `ParquetReader` from parquet-hadoop.

Reply:
Thanks a lot, I fixed this in the last commit and it looks much better!